src/bundle/Resources/public/js/scripts/admin.contenttype.selection.js

```diff
@@ -15,12 +15,21 @@
 
             container.querySelector(SELECTOR_BTN_REMOVE)[methodName]('disabled', disabledState);
         };
+        const getNextId = () => {
+            if (countExistingOptions() === 0) {
+                return 0;
+            }
+            const lastInput = container.querySelector(SELECTOR_OPTIONS_LIST).lastElementChild.querySelector('input[type="text"]');
+            const lastId = lastInput.dataset.ezselectionOptionId;
+
+            return lastId + 1;
```

Comment:
if the `ezselectionOptionId` is '3' this will return '31'. I believe it's not what you want ;)

Reply:
Don;t know how I missed parseInt, it was here before :D